README.md

```diff
@@ -0,0 +1,163 @@
+ti-auth.js
+==========
+
+`ti-auth.js` is a small wrapper to integrate modules into the [Travel Intelligence Portal](https://travel-intelligence.com) providing:
+- **Authentication** - unauthorized users will be redirected to the TI Portal login page and will return after successful authentication.
+- **User Context** - user information, preferences and grants are made available to the application.
+- **Utility functions** - for signout and reauthorization.
+
+
+## Installation
+via Bower:
+
+    $ bower install git@github.com:travel-intelligence/ti-auth.js --save
+
+or via NPM:
+
+    $ npm install https://github.com/travel-intelligence/ti-auth.js --save-dev
+
+## Quick Start
+
+For ease of use the library exposes one global JavaScript object called **TiAuth**.
+
+After including the library in the in your application’s index.html file only 2 simple steps are needed to use `ti-auth.js` for authentication:
+
+1. **Configuration** - depending on the target environment (production/staging/local etc.) the Travel Intelligence API and the
+Portal URL need to be set. This can be done anywhere **before** the first call of the `TiAuth.initialize()`:
+
+    ```javascript
+    TiAuth.API_URL = 'https://api.travel-intelligence.com';
+    TiAuth.DASHBOARD_URL = 'https://portal.travel-intelligence.com';
+    ```
+
+2. **Use the `initialize` method** - to bootstrap your application:
+
+    ```javascript
+    TiAuth.initialize(function(token, user) {
+      //bootstrap your application here
+    });
+    ```
+
+  `token` is the authentication token needed to call Travel Intelligence services
+
+  `user` contains the user with its preferences
+
+## Logout
+
+`ti-auth.js` provides the ability to invalidate the current user session. It will automatically redirect the user to the login page.
+
+```javascript
+TiAuth.logout();
+```
+
+## Framework Integration
+`ti-auth.js` is build to be independent from any specific framework. This enables either a stand-alone use or the integration with common JavaScript frameworks.
+
+### Ember
+
+The initialization can be done in ember by creating a custom initializer.
+
+1. Import `ti-auth.js` in `ember-cli-build.js`:
+ 
+  ```javascript
+  app.import('bower_components/ti-auth.js/dist/ti-auth.min.js');
```

Comment:
Let’s change the path to a more generic one like `'path/to/your/ti-auth.js'` to avoid confusion, *i.e.* when Bower is configured with a different directory or if they use NPM.